quickstart/docker/compose/docker-compose.yaml

```
@@ -18,7 +18,7 @@ services:
       ROUTE_GUIDE_SERVER_PORT: 50051
     volumes:
       - ../../zilla.yaml:/etc/zilla/zilla.yaml
-      - ./proto/route_guide.proto:/proto/route_guide.proto
+      - ./proto/route_guide.proto:/etc/zilla/proto/route_guide.proto
```

```suggestion
      - ./proto:/etc/zilla/proto
```
Suggest we mount the `proto` directory as a whole in docker-compose similar to what we do with helm volume mount.